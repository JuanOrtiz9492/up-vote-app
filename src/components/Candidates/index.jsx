import React from 'react';
import styled from 'styled-components';
import Card from '../Card';
import { constants } from '../../config/constants'

const { size } = constants.photos;
const Container = styled.div`
    display: grid;
    grid-gap: 3em ;
    grid-template-columns: repeat(2, ${size});
    grid-template-rows: repeat(2, ${size});
    @media (max-width: 800px){
        display: flex;
        flex-direction: column;
    }
`

const Candidates = ({ candidates }) => (
    <Container>
        {candidates.map( candidate => ( 
            <Card 
                key={candidate.id} 
                image={candidate.fileName}
                name={candidate.name}
                lastUpdate={candidate.lastUpdate}
                aditionalText={candidate.aditionalText}
            />)
        )}
    </Container>
)

export default Candidates;
