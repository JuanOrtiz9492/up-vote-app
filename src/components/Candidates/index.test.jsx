import React from 'react';
import { render } from '@testing-library/react';
import Candidates from '.';

describe('Candidates Component', () => {

    it('should Match the Snapshot', () => {
        const { container } = render(<Candidates />);
        expect(container).toMatchSnapshot();
      })
})

