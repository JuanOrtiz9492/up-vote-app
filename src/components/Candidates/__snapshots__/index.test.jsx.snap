// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Candidates Component should Match the Snapshot 1`] = `
<div>
  <div
    class="sc-fzXfMy iYcLKJ"
  >
    <div
      class="sc-fzXfMw eyxjsE"
    />
  </div>
</div>
`;
